app/services/provider_authorisation.rb

```diff
@@ -3,6 +3,13 @@ def initialize(actor:)
     @actor = actor
   end
 
+  def can_manage_users?
```

Comment:
I did a quick double take when I saw there was no provider here, but it makes sense under the circumstances. I wonder if it should be `can_manage_users_for_any_providers?`. Then we could have `can_manage_users_for(provider:)`

Reply:
Renamed in https://github.com/DFE-Digital/apply-for-teacher-training/pull/2543/commits/553111bebab0b4bcebaf6074d2f77767676bbca0.